solutions/fully-configurable/main.tf

```diff
@@ -194,6 +195,22 @@ data "ibm_resource_instance" "existing_sm" {
   identifier = var.existing_secrets_manager_crn
 }
 
+module "secrets_manager_group" {
+  depends_on           = [module.secrets_manager]
+  count                = var.create_general_secret_group ? 1 : 0
+  source               = "terraform-ibm-modules/secrets-manager-secret-group/ibm"
+  version              = "1.3.2"
+  region               = local.secrets_manager_region
+  secrets_manager_guid = local.secrets_manager_guid
+  #tfsec:ignore:general-secrets-no-plaintext-exposure
+  secret_group_name        = "General"                        #checkov:skip=CKV_SECRET_6: does not require high entropy string as is static value
+  secret_group_description = "Initially created secret group" #tfsec:ignore:general-secrets-no-plaintext-exposure
+  create_access_group      = var.create_general_secret_group_access_group
+  access_group_name        = "${var.prefix}-General-secrets-group-access-group"
```

Comment:
prefix is optional (it can be null). Ensure to account for this like we do everywhere else prefix is used